tuf/api/metadata.py

```diff
@@ -128,20 +128,40 @@ def from_file(
             A TUF Metadata object.
 
         """
+        if storage_backend is None:
+            storage_backend = FilesystemBackend()
+
+        with storage_backend.get(filename) as file_obj:
+            return cls.from_string(file_obj.read(), deserializer)
+
+    @staticmethod
+    def from_string(
+        data: str,
+        deserializer: Optional[MetadataDeserializer] = None,
+    ) -> "Metadata":
+        """Loads TUF metadata from string.
+
+        Arguments:
+            data: metadata content as string.
+            deserializer: Optional; A MetadataDeserializer instance that
```

Comment:
is this notation standard? Would we be better off just relying on the typing annotations to indicate optional arguments?

Reply:
I saw that in a google style guide example -- could leave that out as well, it does duplicate the annotation...

Reply:
Let's leave it for now. Could you create an issue to ensure we review docstrings and make sure they are consistent before we cut a release.